addons/mail/wizard/mail_compose_message.py

```diff
@@ -1208,7 +1208,10 @@ def _prepare_mail_values_dynamic(self, res_ids):
                 mail_values['attachment_ids'] = process_record._process_attachments_for_post(
                     decoded_attachments,
                     attachment_ids,
-                    {'model': 'mail.message', 'res_id': 0}
+                    {'model': 'mail.message', 'res_id': 0} if (
```

Comment:
double checking while I'm reading this, `res_id = 0` is basically a placeholder for the record that is not yet created? But as model is set, it will still link it at creation?